pkgs/applications/editors/emacs/elisp-packages/update-melpa.el

```diff
@@ -257,10 +257,8 @@ return Promise to resolve in that process."
       url (lambda (status)
             (funcall resolve (condition-case err
                                  (progn
-                                   (goto-char (point-min))
-                                   (search-forward "\n\n")
-                                   (message (buffer-substring (point-min) (point)))
-                                   (delete-region (point-min) (point))
+                                   (message "%S" (url-http-parse-headers))
```

Comment:
A better wording for this commit title:

`update-melpa: use url lib to parse HTTP instead of custom code`